install/helm/README.md

```diff
@@ -27,6 +27,26 @@ The command deploys Agones on the Kubernetes cluster with the default configurat
 
 > **Tip**: List all releases using `helm list`
 
+## Namespaces
+
+By default Agones is configured to work with game servers deployed in the default namespace. If you are planning to use other(s) namespace(s) you can configure Agones via the parameter `gameservers.namespaces`.
+
+For example to use `default` and `xbox` namespaces:
+
+```bash
+$ kubectl create namespace xbox
+$ helm install --set "gameservers.namespaces={default,xbox,ps4}"--name my-release agones
+```
+
+> You need to create your namespaces before installing agones.
```

Comment:
Interesting question - should we create the namespaces for them? Because we could - and it's idempotent, so if they already exist, nothing is going to break. WDYT?

Reply:
I thought about this, wanted to ask on slack your opinion but it was late. I just need to make sure it doesn't block the deployment if the namespace exists.

Reply:
Yeah, good point - if it errors when the namespace already exists, then we shouldn't create the namespace - totally agreed.

Just thinking through some pros and cons otherwise:

Pros:
- Simpler install experience, since you just say what namespaces you want, and Helm will create them for you (if they don't already exist)

Cons:
- If you typo a namespace, you won't know it's broken, because the namespace will be created.

Can you think of any others? At least given the above, I'm leaning towards creating the namespaces, for the easier install experience. It's a bit more "turn key". WDYT?

Reply:
To capture our chat - keep things simple for now, and not implement this. Let's wait on feedback, as there is complexity with helm potentially owning the namespace.

Reply:
the best option if we really want this is to use a pre-install and pre-upgrade hook helm. The hook will create namespace if they don't exist so the namespace is not tracked by helm (once not deleted when the release is deleted, could be dangerous), but that means the hook job needs permissions to create namespace, which complicate things, I also prefer to keep things simple and take action if a real need appears.